docs/open-source/index.mdx

```diff
@@ -1,8 +1,18 @@
 ---
 title: Open source
-description: TODO - filler text so that the SEO check doesn't complain!
+description: Qiskit open-source introduction
```

Comment:
Needs to be >= 50 and <= 160 chars. Abby M and Abby C probably have better suggestions, but maybe:

```suggestion
description: Learn how to get involved with the Qiskit open-source community to shape the future of quantum computing.
```

Reply:
I'm good with this but @javabster is the true expert :-)